src/components/StepIndicator/StepIndicator.js

```diff
@@ -19,12 +19,12 @@ const Step = props => {
   // Filtering out unwanted props.
   const stepProps = {
     className: classes,
-    current: props.current || null,
-    complete: props.complete || null,
-    description: props.description || null,
-    disabled: props.disabled || null,
-    invalid: props.invalid || null,
-    label: props.label, // required
+    current: props.current,
+    complete: props.complete,
+    description: props.description,
+    disabled: props.disabled,
+    invalid: props.invalid,
+    label: props.label,
```

Comment:
setting these to `null` is not necessary because there are default props already